openwebstart/src/main/java/com/openwebstart/jvm/ui/RuntimeManagerPanel.java

```diff
@@ -96,7 +95,7 @@ public RuntimeManagerPanel(final DeploymentConfiguration deploymentConfiguration
     }
 
     private void onRefresh() {
-        backgroundExecutor.execute(() -> {
+        getDaemonExecutorService().execute(() -> {
```

Comment:
It won't be a problem to have this one as a non daemon executor. I prefer to change as less behaviour as possible with this PR so we should change it here to non daemon.